SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -370,6 +370,31 @@ ESchemaComponentType USpatialClassInfoManager::GetCategoryByComponentId(Worker_C
 	return ESchemaComponentType::SCHEMA_Invalid;
 }
 
+const FRPCInfo& USpatialClassInfoManager::GetRPCInfo(UObject* Object, UFunction* Function)
+{
+	check(Object != nullptr && Function != nullptr);
+
+	const FClassInfo& Info = GetOrCreateClassInfoByObject(Object);
+	const FRPCInfo* RPCInfoPtr = Info.RPCInfoMap.Find(Function);
+
+	// We potentially have a parent function and need to find the child function.
+	// This exists as it's possible in blueprints to explicitly call the parent function.
```

Comment:
I'm not sure what this means.

Reply:
It may happen if the function is overloaded in blueprints.

Reply:
I mean I find this comment hard to read. I'm not sure what it's trying to tell me.

Reply:
You can override a blueprint RPC, but you can still call the parent blueprint's RPC, in which case it won't be found in the child blueprint's RPCInfoMap. So we just find a function with the same name because there can (and should) only be one.

Reply:
Discuss offline.